sources/packages/api/src/database/entities/disbursement-schedule.model.ts

```diff
@@ -26,7 +26,7 @@ export class DisbursementSchedule extends RecordDataModel {
    */
   @Column({
     name: "document_number",
-    nullable: false,
+    nullable: true,
   })
   documentNumber: number;
```

Comment:
please add ?  `documentNumber?: number;`, as it can be nullable now.

Reply:
I thought with ? it means the attribute can be an optional field and also undefined